autogpt_platform/backend/backend/integrations/creds_manager.py

```diff
@@ -196,8 +197,25 @@ async def _get_provider_oauth_handler(provider_name_str: str) -> "BaseOAuthHandl
     if provider_name not in HANDLERS_BY_NAME:
         raise KeyError(f"Unknown provider '{provider_name}'")
 
-    client_id = getattr(settings.secrets, f"{provider_name.value}_client_id")
-    client_secret = getattr(settings.secrets, f"{provider_name.value}_client_secret")
+    provider_creds = CREDENTIALS_BY_PROVIDER[provider_name]
```

Comment:
I see it wasn't introduced in this PR but this naming is mildly confusing because now we have two things called "credentials" in the system